docs/manual/guides/deployer.de.md

```diff
@@ -176,6 +177,14 @@ before('deploy:publish', 'contao:manager:download');
 after('contao:manager:download', 'contao:manager:lock');
 ```
 
+Verwendest du Contao Manager nicht, dann kannst du den leeren Ordner von "Shared folders" ausschließen:
```

Comment:
Even if the Contao manager is not used, sharing the folder does not hurt. I'd rather skip this info :-)

Reply:
Sure, doesn't hurt, but I don't like the idea of extra empty folders hanging around in my production servers :) Contao does not come with Manager by default, so why should I have empty `contao-manager` folder?

Reply:
I agree with @richardhj. It would be weird for this guide to use the Contao recipe - only to then remove things that the recipe already provides. Any advanced user will customize this anyway.

Reply:
Ok, I'll remove this.

Reply:
> It would be weird for this guide to use the Contao recipe - only to then remove things that the recipe already provides.

I feel weird that the recipe creates unused files and folders😁 but that question does not belong in this repository, I agree.

Reply:
That's just how recipes work. I do not use any recipes for my deployer Scripts (except `recipe/common.php`) as they all do things that I do not want ;)